### PR TITLE
Update tvtv.us_us.channels.xml

### PR DESCRIPTION
**Pull request by Central302**
Re-try 3, Add ION Television Philadelphia and 3 Internet Channels.

### CODE REVIEW

File: sites/tvtv.us/tvtv.us_us.channels.xml [hunk 1]
```diff
@@ -1234,6 +1234,7 @@
     <channel lang="en" xmltv_id="PlutoTVGameShowCentral.us" site_id="116474">Game Show Central (Pluto)</channel>
     <channel lang="en" xmltv_id="PlutoTVNick.us" site_id="121301">Nick on PlutoTV</channel>
     <channel lang="en" xmltv_id="PlutoTVSPORTBeINSportsXtra.us" site_id="113143">beIN Sports Xtra (English-USA)</channel>
+    <channel lang="en" xmltv_id="PlutoTVBlackNewsChannel.us" site_id="116018">BNCGO (Pluto)/channel>
```
Comment (freearhey) [inline sites/tvtv.us/tvtv.us_us.channels.xml:1237]:
```suggestion
    <channel lang="en" xmltv_id="PlutoTVBlackNewsChannel.us" site_id="116018">BNCGO (Pluto)</channel>
```

File: sites/tvtv.us/tvtv.us_us.channels.xml [hunk 2]
```diff
@@ -1327,6 +1328,7 @@
     <channel lang="en" xmltv_id="StarzKidsFamilyEast.us" site_id="50671">Starz Kids &amp; Family East</channel>
     <channel lang="en" xmltv_id="StarzWest.us" site_id="12741">Starz West</channel>
     <channel lang="en" xmltv_id="Studiocanal.fr" site_id="35751">StudioCanal</channel>
+    <channel lang="en" xmltv_id="SkyNewsInternational.uk" site_id="114448">Sky News for US Streaming/channel>
```
Comment (freearhey) [inline sites/tvtv.us/tvtv.us_us.channels.xml:1331]:
```suggestion
    <channel lang="en" xmltv_id="SkyNewsInternational.uk" site_id="114448">Sky News for US Streaming</channel>
```

File: sites/tvtv.us/tvtv.us_us.channels.xml [hunk 3]
```diff
@@ -1635,6 +1637,12 @@
     <channel lang="en" xmltv_id="WPIXDT2.us" site_id="50777">Antenna TV (WPIX-DT2) NY</channel>
     <channel lang="en" xmltv_id="WPIXDT3.us" site_id="65092">Court TV (WPIX-DT3) NY </channel>
     <channel lang="en" xmltv_id="WPIXDT4.us" site_id="70260">Rewind TV (WPIX-DT4) NY</channel>
+    <channel lang="en" xmltv_id="WPPXDT1.us" site_id="43971">Ion Television (WPPX-DT1) Philadelphia PA/channel>
+    <channel lang="en" xmltv_id="WPPXDT2.us" site_id="43973">Bounce (WPPX-DT2) Philadelphia PA/channel>
+    <channel lang="en" xmltv_id="WPPXDT3.us" site_id="43975">Grit (WPPX-DT3) Philadelphia PA/channel>
+    <channel lang="en" xmltv_id="WPPXDT4.us" site_id="43977">Truereal (WPPX-DT4) Philadelphia PA/channel>
+    <channel lang="en" xmltv_id="WPPXDT5.us" site_id="82712">Laff (WPPX-DT5) Philadelphia PA/channel>
+    <channel lang="en" xmltv_id="WPPXDT6.us" site_id="83522">Defy TV (WPPX-DT6) Philadelphia PA/channel>
```
Comment (freearhey) [inline sites/tvtv.us/tvtv.us_us.channels.xml:1645]:
```suggestion
    <channel lang="en" xmltv_id="WPPXDT1.us" site_id="43971">Ion Television (WPPX-DT1) Philadelphia PA</channel>
    <channel lang="en" xmltv_id="WPPXDT2.us" site_id="43973">Bounce (WPPX-DT2) Philadelphia PA</channel>
    <channel lang="en" xmltv_id="WPPXDT3.us" site_id="43975">Grit (WPPX-DT3) Philadelphia PA</channel>
    <channel lang="en" xmltv_id="WPPXDT4.us" site_id="43977">Truereal (WPPX-DT4) Philadelphia PA</channel>
    <channel lang="en" xmltv_id="WPPXDT5.us" site_id="82712">Laff (WPPX-DT5) Philadelphia PA</channel>
    <channel lang="en" xmltv_id="WPPXDT6.us" site_id="83522">Defy TV (WPPX-DT6) Philadelphia PA</channel>
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by Central302** [comment]
Thanks for Suggest @freearhey !

**Review by freearhey — APPROVED** [review]

**Pull request merged**